app/src/main/res/drawable/continent_europe.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:viewportWidth="423.7945"
+    android:viewportHeight="337.6692"
+    android:width="423.79449dp"
+    android:height="337.66916dp">

```

Comment:
These should be `64dp` if I remember correctly. Bonus points for using `64px` in inkscape already and centering the drawing in it.